uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -57,20 +84,68 @@
  * @TODO remove this patch when Shoreditch theme is migrated to SSP
  */
 .chr_manage_leave_requests {
-  .btn.ui-select-toggle {
-    padding: 4px 10px !important;
-    text-transform: none !important;
+  &[data-role="admin"] {
+    .ui-select-toggle {
+      padding: 4px 10px !important;
+      text-transform: none !important;
+      box-shadow: none !important;
+    }
+
+    .form-group {
+      overflow: inherit !important;
+    }
+
+    .panel {
+      background-image: none !important;
+    }
+
+    .panel-body, .chr_vertical_tabs {
+      background-color: #f4f7f8;
+    }
+
+    input[type="search"] {
+      width: 100% !important;
+    }
+
+    .ui-select-toggle .btn-link i {
+      margin-top: 6px;
+    }
   }
 
-  .form-group {
-    overflow: inherit !important;
+  .btn-link {
+    border: none !important;
+    background-color: transparent !important;
   }
 
-  .panel {
-    background-image: none !important;
+  .chr_manage_leave_requests__sub-header .checkbox {
+    padding-top: 5px;
   }
 
-  .panel-body, .chr_vertical_tabs {
-    background-color: #f4f7f8;
+  .btn-secondary-outline {
+    color: #4D5663;
```

Comment:
@igorpavlov I had a discussion with @AkA84 . Here is the outcome. We should import all the outline classes from shoreditch to SSP. But we cant do that directly because `org.civicrm.shoreditch/scss/bootstrap/overrides/style/_buttons.scss` also contains the some styles which are not related to outlined buttons. So we need to take the code related to only outlined classes and move that to a new folder(may be called "variant/extension") . Then we can import that file directly to SSP without worrying about breaking anything. 
Also this would be a better structure in shoreditch, because right now in overrides folder, we are also including the new variants (like outlined buttons), but it should go to a new folder.

So please create a new folder under `scss/bootstrap/extensions/_outlined-buttons.scss`. And import the same from SSP.
Note: the Shoreditch changes should go to 1.6 and then synced.